### PR TITLE
update changelog for 3.4.0 release

### PR DESCRIPTION
**Pull request by danrosenthal**
### Enhancements

 - Moved icons to a separate npm package ([#686](https://github.com/Shopify/polaris-react/pull/686))
- Added `oneHalf` and `oneThird` props to `Layout` component ([#724](https://github.com/Shopify/polaris-react/pull/724))
- Added `helpText` prop to `ActionList` items ([#777](https://github.com/Shopify/polaris-react/pull/777))
- Updated `Page` header layout so actions take up less room on small screens ([#707](https://github.com/Shopify/polaris-react/pull/707))
- Added `alternateTool` prop to `ResourceList` component ([#812](https://github.com/Shopify/polaris-react/pull/812))
- Updated color of warning status `ExceptionList` items from dark orange to dark yellow for better differentiation from critical status items ([#813](https://github.com/Shopify/polaris-react/pull/813))

 ### Bug fixes

 - Fixed `TextField` not showing the correct color while it has focus and an error ([#806](https://github.com/Shopify/polaris-react/pull/806))
- Fixed `ResourceList` not rendering `BulkActions` on initial load when items were selected ([#746](https://github.com/Shopify/polaris-react/pull/746))
- Fixed the new variant of the `Badge` component so that it is simpler and easier to read ([#751](https://github.com/Shopify/polaris-react/pull/751))
- Reverted a change that set the `autocomplete` property on `TextField` to `nope` when it was `false` ([#761](https://github.com/Shopify/polaris-react/pull/761))
- Added dismiss button for `CalloutCard` ([#353](https://github.com/Shopify/polaris-react/issues/353))
- Removed an extra tab stop from `ResourceList.Item` and make it unactionable while loading ([#745](https://github.com/Shopify/polaris-react/pull/745))
- Fixed `Checkbox` from losing focus when quickly toggled ([#717](https://github.com/Shopify/polaris-react/pull/717))
- Fixed the console error in the `PositionedOverlay` test environment ([#758](https://github.com/Shopify/polaris-react/pull/758))
- Fixed `ResourceList` not rendering a header after initial load (thanks to [@andrewpye](https://github.com/andrewpye) for the [original issue](https://github.com/Shopify/polaris-react/issues/735))
- Fixed `TextField` not passing `step` to the input ([#829](https://github.com/Shopify/polaris-react/pull/829))
- Renamed `Option` and `Group` types in `Select` to work around typedoc oddness ([#830](https://github.com/Shopify/polaris-react/pull/830))

 ### Documentation

 - Modified image paths to fit the [style guide](https://polaris.shopify.com)’s new Markdown parsing rules ([#753](https://github.com/Shopify/polaris-react/pull/753))

 ### Development workflow

 - Added a slight delay to the Percy screenshot script to give time for components to render fully ([#704](https://github.com/Shopify/polaris-react/pull/704))
- Refactors to remove cyclical type imports ([#759](https://github.com/Shopify/polaris-react/pull/759), [#754](https://github.com/Shopify/polaris-react/pull/754), and [#767](https://github.com/Shopify/polaris-react/pull/767))

 ### Dependency upgrades

 - Upgraded `@shopify/polaris-tokens` to v2.1.1 ([#813](https://github.com/Shopify/polaris-react/pull/813))

 ### Code quality

 - Improve `withAppProvider`, `withSticky`, `withRef`, and `withContext` types to allow them to infer a component’s props ([#805](https://github.com/Shopify/polaris-react/pull/805))

### CODE REVIEW

**Comment by danrosenthal** [comment]
https://github.com/Shopify/polaris-styleguide/pull/2488

**Comment by danrosenthal** [comment]
https://github.com/Shopify/web/pull/9544

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,41 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 3.4.0 - 2019-01-07
+
+### Enhancements
+
+- Moved icons to a separate NPM package ([#686](https://github.com/Shopify/polaris-react/pull/686))
```
Comment (kaelig) [inline CHANGELOG.md]: NPM -> npm

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,41 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Modified image paths to fit the [style guide](https://polaris.shopify.com)’s new Markdown parsing rules ([#753](https://github.com/Shopify/polaris-react/pull/753))
+
+### Development workflow
+
+- Added a slight delay to the Percy screenshot script to give time for components to render fully ([#704](https://github.com/Shopify/polaris-react/pull/704))
+- Refactors to remove cyclical type imports ([#759](https://github.com/Shopify/polaris-react/pull/759), [#754](https://github.com/Shopify/polaris-react/pull/754), and [#767](https://github.com/Shopify/polaris-react/pull/767))
+
+### Code quality
+
+- Improve `withAppProvider`, `withSticky`, `withRef`, and `withContext` types to allow them to infer a component's props ([#805](https://github.com/Shopify/polaris-react/pull/805))
```
Comment (kaelig) [inline CHANGELOG.md]: ' -> ’

**Comment by danrosenthal** [comment]
Updated: https://github.com/Shopify/polaris-styleguide/pull/2494

**Pull request merged**